src/main/java/com/parasoft/parabank/service/ParaBankService.java

```diff
@@ -52,6 +52,28 @@
 @WebService(targetNamespace = ParaBankServiceConstants.TNS)
 public interface ParaBankService extends ParaBankServiceConstants {
 
+    /**
+     * Pay bill using funds from the given account
+     *
+     * @param accountId
+     *            the account to which to pay the bill
+     * @param amount
+     *            bill amount
+     * @return status message of result
+     * @throws ParaBankServiceException
+     */
+    @POST
+    @Path("/billpay")
+    @ApiOperation(value = "Pay bill", tags = { ParaBankServiceConstants.ACCOUNTS })
+    @WebResult(name = "billpayReturn", targetNamespace = ParaBankServiceConstants.TNS)
+    @Produces({ MediaType.APPLICATION_XML, MediaType.APPLICATION_JSON })
+    @Consumes({ MediaType.APPLICATION_XML, MediaType.APPLICATION_JSON })
```

Comment:
It would make it easier to "Try it out" in Swagger if APPLICATION_JSON was the default request content type.  Try listing JSON before XML in the Consumes annotation.

Reply:
that worked for consume, do you want me to change it for produces as well

Reply:
No, let's keep the Produces annotation consistent with all the other Swagger resources.